src/main/java/winston/Task.java

```diff
@@ -0,0 +1,71 @@
+package winston;
+
+/**
+ * Represents the tasks that the user input
+ */
+public class Task {
+    private final String description;
+    private boolean done;
```

Comment:
Should follow boolean naming convention
```suggestion
    private boolean isDone;
```